plugins/processors/t128_transform/t128_transform.go

```diff
@@ -153,6 +161,13 @@ func (r *T128Transform) Apply(in ...telegraf.Metric) []telegraf.Metric {
 		}
 	}
 
+	if r.PersistTo != "" {
```

Comment:
We should be able to do this more intelligently. Do we not know specifically when the cache has changed?

Reply:
tried reflect.DeepEqual but that didn't work so went with an inline comparison instead (which is probably faster anyways)